app/components/newscarousel.js

```diff
@@ -58,6 +58,7 @@ const Item = (props) => {
           height={140}
           src={props.item.attributes.imageUrl}
           alt={props.item.attributes.name}
+          className={`${styles.imgItm}`}
```

Comment:
Thanks for the PR. Just a minor suggestion. In `className`, you can also define it as `className={styles.imgItm}`.

Reply:
Thank you for your suggestion @Dnouv

Reply:
LGTM! Thanks for the quick changes.